lib/financials.js

```diff
@@ -104,17 +104,30 @@ async function getFinancialDataFromServer(): Promise<FinancialDataShapeType> {
     AsyncStorage.setItem(FINANCIALS__CACHE__BALANCES, JSON.stringify(new Date())),
   ])
 
-  return {flex, ole, print}
+  return {flex, ole, print};
 }
 
 
 // TODO: come up with a better story around auth for olecard
-export default async function getWeeklyMealsRemaining() {
+export async function getWeeklyMealsRemaining() {
   let {username, password} = await loadLoginCredentials()
   let form = buildFormData({
     username: username,
     password: password,
   })
-  let loginResult = await api.post('/apps/olecard/checkbalance/authenticate.cfm', {body: form})
-  return loginResult
+  let fetchResult = await api.post('/apps/olecard/checkbalance/authenticate.cfm', {body: form})
+
+  try {
+    let dom = parseHtml(fetchResult.body)
+    let data = cssSelect('.accountrow', dom)
+    let values = []
+    data.map((item) => {
+      values.push(item.next.next.children[0].data)
+    })
```

Comment:
Please don't use `map` for side-effects; `map` should be a pure function, not mutating anything outside of its scope.

Instead, you could do 

```js
let values = data.map((item) => {
  return item.next.next.children[0].data
})
```